types/parse/index.d.ts

```diff
@@ -832,6 +832,13 @@ declare global {
                 sort?: { [key: string]: 1 | -1 } | undefined;
                 // Sample documentation: https://docs.mongodb.com/v3.2/reference/operator/aggregation/sample/
                 sample?: { size: number } | undefined;
+                count?: string | undefined;
```

Comment:
Count is a string??

Reply:
Yes - it actually defines the name of the field that is used to store the count value as part of the response. Have updated with a link to the docs